src/telegram_to_engagement_db/telegram_group_to_engagement_db.py

```diff
@@ -119,8 +120,14 @@ def _telegram_message_to_engagement_db_message(telegram_message, dataset, uuid_t
     :return: `telegram_message` as an engagement db message.
     :rtype: engagement_database.data_models.Message
     """
-    participant_uuid = uuid_table.data_to_uuid(telegram_message.sender_id)
-    channel_operator = 'telegram'  # TODO move to core as a CONSTANT
+
+    # Format the telegram id to match rapid pro format "telegram:123456" and
+    # Sometimes a telegram urn ends with an optional #<username> e.g. telegram:123456#testuser
```

Comment:
I wonder if we should put this in core, along with the similar code in the rapid pro -> db stage, in e.g. a normalise_urn function